RxCocoa/iOS/UIScrollView+Rx.swift

```diff
@@ -77,6 +77,21 @@
             return ControlEvent(events: source)
         }
         
+        /// Reactive wrapper for delegate method `scrollViewWillEndDragging(_:withVelocity:targetContentOffset:)`
+        public var willEndDragging: ControlEvent<(CGPoint, UnsafeMutablePointer<CGPoint>)> {
```

Comment:
Oh, I've noticed one more thing.
Could you please make this `public var willEndDragging: ControlEvent<(velocity: CGPoint, targetContentOffset: UnsafeMutablePointer<CGPoint>)> `.